advanced/editor-command-identifiers.md

```diff
@@ -7,93 +7,465 @@ description: The complete list of exposed editor commands.
+tinymce.activeEditor.execCommand('mceRemoveTextcolor', 'hilitecolor')
+tinymce.activeEditor.execCommand('RemoveFormat')
+tinymce.activeEditor.execCommand('mceBlockQuote')
+tinymce.activeEditor.execCommand('FormatBlock', false, 'bold')
+tinymce.activeEditor.execCommand('mceInsertContent', false, 'My new content')
+tinymce.activeEditor.execCommand('mceReplaceContent', false, 'My replacement content')
+tinymce.activeEditor.execCommand('mceSetContent', false, 'My content')
+tinymce.activeEditor.execCommand('mceToggleFormat', false, 'bold')
+tinymce.activeEditor.execCommand('ToggleSidebar')  /* OR */
+tinymce.activeEditor.execCommand('ToggleSidebar', false, '<sidebar-name>')
+tinymce.activeEditor.execCommand('Indent')
+tinymce.activeEditor.execCommand('Outdent')
+tinymce.activeEditor.execCommand('InsertHorizontalRule');
+tinymce.activeEditor.execCommand('InsertLineBreak')
+tinymce.activeEditor.execCommand('mceInsertNewLine')
+tinymce.activeEditor.execCommand('mceInsertRawHTML', false, '<p>Hello, World!</p>')
+tinymce.activeEditor.execCommand('mceToggleVisualAid')
+tinymce.activeEditor.execCommand('SelectAll')
+tinymce.activeEditor.execCommand('Delete')
+tinymce.activeEditor.execCommand('ForwardDelete')
+tinymce.activeEditor.execCommand('mceNewDocument')
+tinymce.activeEditor.execCommand('Redo')
+tinymce.activeEditor.execCommand('Undo')
+tinymce.activeEditor.execCommand('mceAddUndoLevel')
+tinymce.activeEditor.execCommand('mceEndUndoLevel')
+tinymce.activeEditor.execCommand('mceCleanup')
+tinymce.activeEditor.execCommand('mceSelectNode', false, '<DOM_node>')
+tinymce.activeEditor.execCommand('mceSelectNodeDepth', false, 2) // For two nodes up.
+tinymce.activeEditor.execCommand('mceRemoveNode') /* OR */
+tinymce.activeEditor.execCommand('mceRemoveNode', false, '<DOM_node>')
+```
+
+### FormatBlock and mceToggleFormat format options
+
+The following options can be used with the `FormatBlock` and `mceToggleFormat` commands.
+
+**General format options**
+
+The following options will work on most content.
+
+| Format option  | Description                                                                                                                         |
+| -------------- | ----------------------------------------------------------------------------------------------------------------------------------- |
+| alignleft      | Toggles left alignment of the selected content.                                                                                     |
+| aligncenter    | Toggles center alignment of the selected content.                                                                                   |
+| alignright     | Toggles right alignment of the selected content.                                                                                    |
+| alignjustify   | Toggles 'justified' alignment of the selected content.                                                                              |
```

Comment:
Should the quotes be around justifed here, given we don't have it in any other alignment descriptions?

Reply:
fixed